python-sdk/examples/fetch_policies_demo.py

```
@@ -0,0 +1,185 @@
+    langwatch.setup(debug=True)
+
+    start_time = time.time()
+    prompt = langwatch.prompts.get(prompt_name)  # Uses default MATERIALIZED_FIRST
+    end_time = time.time()
+
+    print(f"   Time taken: {end_time - start_time:.3f} seconds")
+    print(f"   Model: {prompt.model}")
+    print("   ✅ Success - loaded from local (no API call needed)")
+
+
+def demo_always_fetch(work_dir: Path, prompt_name: str):
```

Ruff (0.14.8)

Unused function argument: `prompt_name`

(ARG001)